src/Corporation/Industry.ts

```diff
@@ -838,6 +838,7 @@ export class Industry implements IIndustry {
                   //Dynamically evaluated
                   let tmp = (mat.sllman[1] as string).replace(/MAX/g, (maxSell + "").toUpperCase());
                   tmp = tmp.replace(/PROD/g, mat.prd + "");
+                  tmp = tmp.replace(/STORE/g, (mat.qty / (CorporationConstants.SecsPerMarketCycle * marketCycles)) + "");
```

Comment:
is this bonus time safe too?

Reply:
doesn't bonus time just reduce the time between market cycles? tbh, the code here is pretty hard to read.

Reply:
yeah tbh i have no idea, theres a reason this code causes so many bugs ;D